app/styles/main.scss

```diff
@@ -93,9 +97,43 @@
   }
 }
 
-.footer {
-  margin-top: 1em;
-  .bar-right {
-    margin: 30px;
+.footer > .bar-right {
+  margin: 30px;
+}
+
+.box-login {
+  max-width: 400px;
+  margin: auto;
+  display: flex;
+  flex-direction: column;
+
+  input, button {
+    margin: 10px 0;
+    height: 35px;
+    font-size: 1.2em;
+  }
+}
+
+.btn-accent {
+  background-color: $color-accent-light;
+  border: 1px solid $color-accent-light;
+  border-bottom: 3px solid $color-accent;
+  border-radius: 5px;
+  color: white;
+}
+
+.hermes {
```

Comment:
Move this to somewhere at the beginning of this file since it seems to be applying to a top level element.